contrib/babelfishpg_tsql/sql/sys_views.sql

```diff
@@ -1283,17 +1283,31 @@ select
       when 'p' then 'P'::varchar(2)
       when 'a' then 'AF'::varchar(2)
       else
-        case format_type(p.prorettype, null) when 'trigger'
-          then 'TR'::varchar(2)
+        case 
+          when pg_catalog.format_type(p.prorettype, null) = 'trigger'
+            then 'TR'::varchar(2)
+          when p.proretset then
+            case 
+              when t.typtype = 'c'
```

Comment:
From what I've observed, it seems like Table type function will always return a composite type in pg_types, but inline-table-valued functions will return record or another datatype (like int in the case that it returns a single column)